proposals/jib_core_library.md

```diff
@@ -31,46 +38,67 @@ Design for Jib Core as a Java library for building container images.
 Three `TargetImage` types (`RegistryImage`, `DockerDaemonImage`, and `TarImage`) define the 3 different targets Jib can build to.
 
 `RegistryImage` - builds to a container registry
+
 - `static RegistryImage named(ImageReference/String)`
 - `RegistryImage addCredential(String username, String password)`
 - `RegistryImage addCredentialRetriever(CredentialRetriever)`
 
 `DockerDaemonImage` - builds to a Docker daemon
+
 - `static DockerDaemonImage named(ImageReference/String)`
 - `DockerDaemonImage setDockerExecutable(Path)`
 
 `TarImage` - builds to a tarball archive
+
 - `Builder`
   - `TarImage saveTo(Path outputFile)`
 - `static Builder named(ImageReference/String)`
 
 `Containerizer` - configures how and where to containerize to
-- `static Containerizer to(RegistryImage)`
-- `static Containerizer to(DockerDaemonImage)`
-- `static Containerizer to(TarImage)`
 - `Containerizer setExecutorService(ExecutorService)`
 - `Containerizer setCacheConfiguration(CacheConfiguration)`
 - `Containerizer setEventHandlers(EventHandlers)`
 
-## For Java containers
+## Simple example
+
+```java
+Jib.from("busybox")
+   // TODO: this doesn't seem right, especially if multiple files are in the .addLayer call (are they cat'ed, or is /helloworld.sh a directory?)
```

Comment:
The `pathInContainer` is actually a directory path to place the files under. Maybe it makes more sense to call it `directoryInContainer`?

Reply:
directoryInContainer is good. Updating the example...